eclog/src/test/resources/ecLog.yaml

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
Would it make sense to add a sample config file with some descriptions in the /conf directory as well?

Reply:
Yes, I'll do that!